src/test/functional/features/joint-rfi-response.feature

```diff
@@ -0,0 +1,249 @@
+Feature: Joint request for information
+
+  Background: Logged in for hub page
+    Given I create a new user and login
+    And I've already completed the form using the fixture "jointApplicant1CompleteCase"
+    When I go to "/"
+    And I click "Send for review"
+    Then the page URL should be "/application-sent-for-review"
+    When I enter my valid case reference and valid access code
+    Then the page should include "You need to review your joint application"
+    Given I've already completed the form using the fixture "jointApplicant2CompleteCase" for "applicant2"
+    And I go to "/applicant2/confirm-your-joint-application"
+    Given I select "I confirm that I’m applying to the court to dissolve my marriage (get a divorce)"
+    And I select "I believe that the facts stated in this application are true"
+    When I click "Submit"
+    Then the page URL should be "/applicant2/needs-to-confirm-joint-application"
+
+  @nightly
+  Scenario: [1] Applicant 1 joint rfi journey
+    Given a case worker issues a request for information to app1 on a joint case
+    When I click "Sign out"
+    And I login with applicant "2"
+    Then the page URL should be "/applicant2/hub-page"
+    And the page should include "The court has reviewed your application for divorce. We have sent an email to your wife with the information that the court needs."
+    And the page should include "The court will review the information from your wife once provided, then the application can progress."
```

Comment:
It might be best to match against shorter strings of text to speed up the tests and make them less brittle. Matching against element IDs is often suggested as the best practise because they don't change as often as text, meaning less need to update the test

Reply:
Thanks @BWeeks-JusticeGovUK, if it's easier to shorten the text that should be fine as well